modules/Nets/staNNet.py

```diff
@@ -282,7 +284,11 @@ def load_data(self, data):
 
     def outputs(self,inputs):
         data = self.load_data(inputs)
-        return self.model(data).data.cpu().numpy()[:,0]
+        return self.model(data).data.cpu().numpy()[:,0] * self.info['conversion']
+    
+    def outputs_torch(self,inputs):
```

Comment:
why/where is this needed?

Reply:
the weights of the NN are trained on data where the output is divided by info['conversion']. To make sure the NN will always give outputs in terms of nA, we need to multiple whatever comes out of self.model with this conversion factor.

Reply:
I mean the output_torch function. I guess this is a convienience function, so it can be implemented together with output?

Reply:
outputs_torch function is now merged with outputs function: if grad=False, output will be numpy array, if grad=True output will be a tensor (default is False)